tgui/packages/tgui/webpack.config.js

```diff
@@ -29,6 +29,7 @@ module.exports = (env = {}, argv) => {
           : path.resolve(__dirname, './public/.tmp'),
       filename: '[name].bundle.js',
       chunkFilename: '[name].chunk.js',
+      hashFunction: 'SHA256',
```

Comment:
I think this is the default, so I think this should be fine, just want to mark it down.

Reply:
you'd think so, but without this it fails to run due to an unsupported digest (hash function)